tests/local/component-model/component-model-fixed-size-list/fixed-size-list.wast

```diff
@@ -0,0 +1,38 @@
+(component
+  (core module $m
+    (memory (export "memory") 1)
+    (func (export "ret-list") (result i32) unreachable)
+  )
+  (core instance $i (instantiate $m))
+
+  (func (export "ret-list") (result (list u32 4))
```

Comment:
I'm actually a little confused, how does this test pass? The fixed-size-list is appropriately gated in wasmparser (or so I thought) but I would expect a change to `tests/roundtrip.rs` to enable the fixed-size-list feature here

Reply:
I'm still sort of surprised here though because this test is indeed passing on CI, and it looks like it is indeed testing fixed-size-lists, so I'm not sure why it's passing...

Reply:
Ok nevermind I see that `WasmFeatures::all()` is used in tests, so disregard this.